tests/foreman/cli/test_bootdisk.py

```diff
@@ -0,0 +1,94 @@
+"""Tests for BootdiskPlugin
+
+:Requirement: Bootdisk
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: BootdiskPlugin
+
+:Team: Rocket
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_mac
+from fauxfactory import gen_string
+
+from robottelo.cli.factory import make_medium
+from robottelo.config import settings
+from robottelo.constants import HTTPS_MEDIUM_URL
+
+
```

Comment:
Marking on_prem_provisioning might be appropriate for this test

Reply:
Not needed as we are just checking if the full host bootdisk is downloaded properly for the created host. Bootdisk provisioning is in the plan which I will add in future.